pkg/jx/cmd/init.go

```diff
@@ -493,7 +493,7 @@ func (o *InitOptions) initIngress() error {
 		}
 
 		values := []string{"rbac.create=true" /*,"rbac.serviceAccountName="+ingressServiceAccount*/}
-		valuesFiles := []string{}
+		valuesFiles := []string{"./myvalues.yaml"}
 		if o.Flags.Provider == AWS || o.Flags.Provider == EKS {
```

Comment:
we should probably check if the file exists and also use a non platform file separator, i.e. `filepath.Join(".", "myvalues.yaml")` not `./myvalues.yaml`

Reply:
That makes sense. But I get a bit confused with how it's handled in install.go. First valuesFiles is initialized as:
```go
valueFiles := []string{"./myvalues.yaml", "./secrets.yaml", secretsFileName, adminSecretsFileName, configFileName}
```

Then myvalues.yaml is appended to this list as described by a comment:

	// Overwrite the values with the content of myvaules.yaml files from the current folder if exists, otherwise
	// from ~/.jx folder also only if is present

Is this a mistake and "./myvalues.yaml" should be removed from the initialization?

How about I extract the code implementing the above comment to a separate method and use in both places?